### PR TITLE
Bug 1187688 - After taping on URL bar field and pressing backspace sets character selection instead of deleting it.

### PR DESCRIPTION
**Pull request by codestergit**
Notifying text change after the applying completion.

### CODE REVIEW

File: Client/Frontend/Widgets/AutocompleteTextField.swift [hunk 1]
```diff
@@ -64,6 +64,9 @@ class AutocompleteTextField: UITextField, UITextFieldDelegate {
         if completionActive {
             self.attributedText = NSAttributedString(string: text)
             completionActive = false
+            // This is required to notify the SearchLoader that some text has changed and previous
+            // cached query will get updated.
+            self.autocompleteDelegate?.autocompleteTextField(self, didEnterText: self.text)
```
Comment (thebnich) [inline Client/Frontend/Widgets/AutocompleteTextField.swift]: Can we use `notifyTextChanged?()` here to stay consistent?

**Comment by codestergit** [comment]
@thebnich  
Thanks for suggestion. Updated the pull request.

**Comment by codestergit** [comment]
Hi @thebnich 
Updated the pull request.Thanks

**Comment by thebnich** [comment]
Looks good, thanks!

**Pull request merged**